src/libs/nav2_util/include/nav2_util/sensors/laser.h

```diff
@@ -29,7 +29,7 @@
 #ifndef AMCL_LASER_H
```

Comment:
Have you run the linters on this change? I'm thinking AMCL_LASER_H will fail cpplint.

Reply:
There will be another PR to address all the linter issues.